src/dstr-binding/ary-ptrn-rest-obj-prop-id.case

```diff
@@ -18,6 +18,8 @@ info: |
        [...]
 ---*/
 
+//- setup
+let length = "outer";
```

Comment:
I don't get what would be different using a Symbol value. 

```js
const s = Symbol("outer");
const length = s;

function scope(/* ... */) {
  assert.sameValue(length, s);
}

scope();
```

this would be like the same as checking if the value as an string.

One thing that is interesting in this case, is that we already protect it in the dstr pattern:

```js
[...{ 0: v, 1: w, 2: x, 3: y, length: z }]

/* ... */
assert.sameValue(z, 3);
```

When we assert `z` is 3, we could assume a wrong `length` would be 3, or close. Not `"outer"`.